CMakePresets.json

```diff
@@ -58,6 +60,8 @@
       "cacheVariables": {
         "VCPKG_TARGET_TRIPLET": "x64-windows",
         "BUILD_GUI": "ON",
+        "BUILD_BENCH": "ON",
```

Comment:
Why on for Windows?

Reply:
Thanks!. [Moved](https://github.com/hebasto/bitcoin/pull/196#issuecomment-2126735480) to the CI script.